src/main/java/picard/util/IntervalListTools.java

```diff
@@ -501,6 +514,22 @@ private IntervalList openIntervalLists(final List<File> files, BinaryOperator<In
                 .orElse(null);
     }
 
+    private static IntervalList uniqued(IntervalList non_unique, boolean merge_abutting) {
```

Comment:
why does this need to return an IntervalList? can't you let it return a list of intervals instead?  You immediately call `getIntervals()` on it the only place I see it used, so it seems like a lot of unneeded converting back and forth between lists of intervals and interval lists.

Reply:
Because I'm reproducing the functionality of "uniqued" in htsjdk's IntervalList. The idea is to have a "uniqued" method that we can one day add to their code base in a PR. Alternatively they have a "getUniqueIntervals()" method that rerturns a List of Intervals. I could use that but this makes for cleaner code with all the defaults. 

tl;dr: this was purely a design decision, I don't have a strong preference on which way to do it, and I'm happy to change it to according to your preferences, as you will also be interacting with it.

Reply:
I see, that seems reasonable then